tests/src/junit/resources/META-INF/neoforge.mods.toml

```diff
@@ -6,3 +6,5 @@ license="LGPL v2.1"
 modId="startup_config_test"
 [[mods]]
 modId="creative_tab_order_test"
+[[mods]]
```

Comment:
I propose a standard for these mod IDs: order them alphabetically to reduce the chances of conflicts.